index.html

```diff
@@ -40,6 +40,17 @@
                 </div>
 
                 <div class="row">
+                    <div class="col-lg-12 text-center">
```

Comment:
I meant something like the following -

```
<div class="row">
  <div class="col-lg-3"></div>
  <div class="col-lg-6">
    ...
  </div>
</div>
```

So that the alert box doesn't take up the entire container width and is only there in the center.